corehq/apps/users/static/users/js/mobile_workers.js

```diff
@@ -208,6 +208,7 @@ hqDefine("users/js/mobile_workers",[
         self.useStrongPasswords = ko.observable(options.strong_mobile_passwords);
         self.useDraconianSecurity = ko.observable(options.draconian_security);
         self.implementPasswordObfuscation = ko.observable(options.implement_password_obfuscation);
+        self.twoStageProvisioningEnabled = ko.observable(options.two_stage_provisioning_enabled);
```

Comment:
Nitpick: this doesn't need to be an observable since it isn't going to change.

Reply:
Nevermind, just looked around more and saw the comment above this block.

Reply:
...but it does look like this value is created twice (here and two lines down) and never used.

Reply:
thanks https://github.com/dimagi/commcare-hq/pull/26946/commits/57ca691348420c170649f633bd3b5387a6d5a842